RiotSwiftUI/Modules/Authentication/Registration/View/AuthenticationRegistrationScreen.swift

```diff
@@ -0,0 +1,198 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct AuthenticationRegistrationScreen: View {
+
+    // MARK: - Properties
+    
+    // MARK: Private
+    
+    @Environment(\.theme) private var theme: ThemeSwiftUI
+    
+    // MARK: Public
+    
+    @ObservedObject var viewModel: AuthenticationRegistrationViewModel.Context
+    
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 0) {
+                header
+                    .padding(.top, OnboardingMetrics.topPaddingToNavigationBar)
+                    .padding(.bottom, 36)
```

Comment:
Optional for future PR: might be good to extract these hardcoded padding constants into some internal struct / enum for more control and consistency.

Reply:
I'm slowing pulling out common constants into `OnboardingMetrics`, but I have wondered about local constants too. So far my conclusion was that unless some of these constants are shared, then the benefit is small and is no different to entering constraint values into a xib/storyboard. But definitely something I'd like to talk about more.